setup.py

```diff
@@ -60,6 +60,8 @@
         'ms': ['python-casacore >= 2.2.1'],
         's3': ['botocore'],
         # rados is not in PyPI but available as Debian package python-rados
-        'rados': ['rados']
+        'rados': ['rados'],
+        # Only available throught github currently
+        'auth' : ['katsdpauth'], 
```

Comment:
Sorry to be pedantic, but now you have one space too many :-)

Reply:
Haha, whoops.